src/modules/mod.rs

```diff
@@ -95,8 +103,8 @@ mod tests {
 
     pub(crate) fn get_block() -> db_helpers::Block {
         db_helpers::Block {
-            timestamp: 1655571176644255779,
-            height: 68000000,
+            timestamp: 1670867692546051383, // December 12, 2022
+            height: 80500000,
```

Comment:
I had to re-run all the tests to be able to use the new DB in snapshots. We still haven't the piece I've used in tests previously :(